test/metric_value_benchmark/base_test.go

```diff
@@ -58,25 +60,32 @@ func (t *TestRunner) runAgent() (status.TestGroupResult, error) {
 		},
 	}
 
+	extraCommands := t.testRunner.getExtraCommands()
```

Comment:
this doesn't seem like it belongs to 'runAgent'? It doesn't really have to do with running the agent right..? or did I misunderstand

Reply:
Yup its has nothing to do with agent !!! That's why I kept it separate from agents and be generic as possible.